tauv_common/launch/thruster_managers/thruster_manager.launch

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<launch>
+  <!-- Name of the robot model  -->
+  <arg name="model_name" default="manta"/>
+  <!-- Namespace of the robot in the simulation (it can be different from the original model name) -->
+  <arg name="uuv_name" default="$(arg model_name)"/>
+  <!-- Name of the base link -->
+  <arg name="base_link" default="base_link" />
+  <!--
+    Thruster manager timeout.
+    If no wrench topic is received in the thruster
+    manager input for the time determined by the timeout, the manager will
+    publish zeros.
+    If the timeout is set to -1, the latest received input is always going to
+    be redistributed to the thruster units.
+  -->
+  <arg name="timeout" default="-1" />
+  <!--
+    In case the TAM.yaml must be generated or updated, you can set this flag to
+    true. The thruster allocation matrix is stored in a file because the
+    tf listener, that retrieves the pose of the thrusters wrt the vehicle's
+    base link in runtime, takes a long time to start. This initialization time
+    can make the vehicle's inactive until the tf frames are available. Be
+    sure to keep the thruster allocation matrix stored in a TAM.yaml to
+    improve the initialization of the thruster manager.
+    To generate the TAM.yaml, start the simulation with the vehicle and run:
+
+    >> roslaunch manta_control start_thruster_manager.launch reset_tam:=true
+
+    The TAM.yaml file will be store in the folder defined by the following
+    argument output_dir
+  -->
+  <arg name="reset_tam" default="false"/>
+  <!-- Output folder to store the TAM.yaml and thruster_manager.yaml files -->
+  <arg name="output_dir" default="$(find tauv_common)/config"/>
```

Comment:
yeah, I think the right option is to create a "models" folder in common, then a subdirectory for each vehicle. We can store the TAM and thruster_manager.yaml files there, as well as any other vehicle config files we might want.